semantic/src/site/globals/site.overrides

```diff
@@ -319,6 +329,51 @@ a:hover {
   max-width: 150px;
 }
 
+/*---------------------------------------
+      Review page
+----------------------------------------*/
+
+// NOTE: The review page shares a lot of styles with Application Summary
+
+.review-comment-area {
+  background-color: @reviewCommentBackground;
+  margin-top: 10px;
+  margin-left: -20px;
+  margin-right: -20px;
+  margin-bottom: -14px;
+  border-top: 1px solid @surfacesMed;
+  border-bottom: 1px solid @surfacesMed;
+}
+
+.review-comment-grid.ui.grid {
+  padding: 10px 20px;
+  font-size: 0.85em;
```

Comment:
Should probably use one of the predefined one in `site.variables`

Reply:
You mean font-size? I kind of wanted it to be a relative size here (and we probably should have done a bit more of it). The important thing is that this text is "a bit smaller" than the paragraph text, absolute size is not particularly important. So if we change the paragraph text size, this will automatically change proportionally, which is much easier to manage. We could make some classes that do basic proportioning in `ems` at some point, but might be best to go through once we're ready for a tidy-up. I've got a bunch of little things that can be consolidated, but it's best to wait till the end I think.

Reply:
ok